cp23/hpi-fun2/motivation/problem.tex

```diff
@@ -0,0 +1,56 @@
+\input{template.tex}
+
+\begin{document}
+
+\makeheader
+
+One day, Christian, the CEO of \textit{Super Arrangement Productions}, entered one of their offices
+to talk to an employee who complained about the lack of motivation in their company. 
+Christian was quite surprised, that not only the department, but the whole company was not motivated
+at all.
+
+That is why Christian made the decision to hire a motivation agency to send the best of their coaches
+to help motivate the people again.
+
+In a brief knowledge exchange with the head of the coaches, Christian learned the following things:
+
+\begin{itemize}
+    \item Each employee has a demotivational level, which tells how much effort it needs for a coach to motivate the employee again.
+    \item The agency sends their best coaches with each having a specific motivational level.
+    \item Motivating employees is exhausting, which means that coaches and employees motivating other employees will be less motivated, equally to the level of demotivation.
+    \item Employees are motivated by at least one person, but an employee will not be willing to get motivated by a person that higher hierarchy layers did not get motivated by.
+    \item After being motivated again, employees are willing to motivate other employees, but only if those employees are directly or indirectly under their current position.
```

Comment:
Your problem is quite complex (but great!) and at least to me the task is not clear immediately. So try either:
- simplifying/clarifying your problem statement
- add a formalization
- add some kind of other helpful stuff (like visuals)

I think this will increase the fun with the problem as the problem is indeed very interesting!

Reply:
I tried simplifying it, but I am not sure, if I improved it.